pkgs/applications/audio/easyabc/default.nix

```diff
@@ -16,7 +16,14 @@ python39.pkgs.buildPythonApplication {
 
   propagatedBuildInputs = with python39.pkgs; [
     cx_Freeze
-    wxPython_4_2
+    (wxPython_4_2.overrideAttrs (args: rec {
```

Comment:
please add a comment about why this was necessary

Reply:
done in https://github.com/NixOS/nixpkgs/pull/243787/commits/f0c012c72817f91d500e5d307fc66c5d5a649957